### PR TITLE
chore: 🤖 alias target route

### PR DESCRIPTION
**Pull request by DhariniJeeva**
:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12959)

## Description

This PR adds a subroute within targets for the users to be able to create an alias within a target

## Screenshots (if appropriate):

 

https://github.com/hashicorp/boundary-ui/assets/15043878/a1ee4787-8ab2-441b-bcef-006bfcc362de

mirage: https://boundary-omzeos2yz-hashicorp.vercel.app/

## How to Test
case 1: 
1. Get the latest main from the boundary repo and run the controller
2. create a TCP target with an alias
3. After successful creation, it should be added to the sidebar when you click on the created target
4. clicking on the alias (sidebar) should take you to the actual alias form so the user can edit if they like

case 2: 
1. if you didn't add an alias while creating a target, the sidebar should show an option to create an alias within that target. 

 

:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:

- [x] I have added before and after screenshots for UI changes
- ~[ ] I have added JSON response output for API changes
- ~[ ] I have added steps to reproduce and test for bug fixes in the description
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <Hds::Dropdown as |dd|>
+      <dd.ToggleButton
+        data-test-manage-dropdown
+        @text={{t 'actions.manage'}}
+        @color='secondary'
+      />
+
+      <dd.Interactive
+        @text={{t 'actions.clear'}}
+        {{on 'click' (route-action 'clearAlias' @model)}}
```
Comment (DhariniJeeva) [inline ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs:33]: note: I did not move these to controllers as the target route involves quite a bit of changes

Comment (calcaide) [reply ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs:33]: I was going to suggest for the sake of the code freeze to leave the refactor to controllers for later. We already have a [jira task](https://hashicorp.atlassian.net/browse/ICU-12611) dedicated to this, so all good 😉

**Review by calcaide — APPROVED** [review]

**Review by calcaide — APPROVED** [review]
Great job Dharini!! new alias creation looking good!! 😎

File: ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs (hunk 1 above)
Comment (DhariniJeeva) [reply ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs:33]: thanks, for linking that, I was not aware that we already had tickets for this :D

**Pull request merged**